src/v3/src/components/PasswordRequirements/PasswordMatches.tsx

```diff
@@ -58,16 +57,16 @@ const PasswordMatches: UISchemaElementComponent<{
 
   return (
     <Box data-se="password-authenticator--matches">
-      <List
+      <Box
+        component="ul"
```

Comment:
Removing the usage of the `List` MUI element because in the latest version of Odyssey they are customizing the style that forces us to override them. This component only uses list for semantic reasoning not for the style so just using `Box`.